### PR TITLE
Fix vulnerabilities, remove cpy lib and upgrade to node 21

### PR DESCRIPTION
**Pull request by badiuoanaalexandra**
- Fixed vulnerability related to braces 
- removed cpy lib - cpy had also vulnerability related to braces lib. cpy has many issues and shouldn't be used anymore, so removed it and replaced the functionality with node native libs (fs and path).
- updated to node 21
- tested all scripts and everything works

Since this is just a dev change I didn't update any packages.

### CODE REVIEW

**Review by tristen — APPROVED** [review]
LGTM

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-16
\ No newline at end of file
+21
```
Comment (AlexanderBelokon) [inline .nvmrc:1]: I don't think we should ever use non-LTS Node releases, and all odd ones don't get LTS

Comment (badiuoanaalexandra) [reply .nvmrc:1]: Aaah.. that's annoying.. I didn't know that odd ones don't get it. I'll change to 20

Comment (badiuoanaalexandra) [reply .nvmrc:1]: https://github.com/mapbox/mr-ui/pull/225

**Pull request merged**